cmd/start.go

```diff
@@ -56,16 +58,26 @@ func startNetwork(cmd *cobra.Command, args []string) error {
 	ctx := binutils.GetAsyncContext()
 
 	ux.Logger.PrintToUser(startMsg)
+
+	outputDirPrefix := path.Join(app.GetRunDir(), "restart")
+	outputDir, err := utils.MkDirWithTimestamp(outputDirPrefix)
+	if err != nil {
+		return err
+	}
+
 	loadSnapshotOpts := []client.OpOption{
 		client.WithPluginDir(pluginDir),
 		client.WithExecPath(avalancheGoBinPath),
+		client.WithRootDataDir(outputDir),
 	}
 	_, err = cli.LoadSnapshot(
 		ctx,
 		snapshotName,
 		loadSnapshotOpts...,
 	)
+
 	if err != nil {
+		// TODO: use error type not string comparison
```

Comment:
Do we have a ticket for this TODO?

Reply:
https://ava-labs.atlassian.net/browse/AV-1749